universum/analyzers/utils.py

```diff
@@ -2,13 +2,28 @@
 import sys
 
 
-def add_common_arguments(parser):
+def add_files_argument(parser):
+    parser.add_argument("--files", dest="file_list", nargs='+', required=True,
+                        help="Python files and Python packages for analyzer. "
+                             "Files could be defined as a single python file"
+                             " *.py or directories with __init__.py file in the directory.")
+
+
+def add_result_file_argument(parser):
     parser.add_argument("--result-file", dest="result_file",
                         help="File for storing json results of Universum run. Set it to \"${CODE_REPORT_FILE}\" "
                              "for running from Universum, variable will be handled during run. If you run this "
                              "script separately from Universum, just name the result file or leave it empty.")
 
 
+def add_python_version_argument(parser):
```

Comment:
But we have other analyzers then pylint and mypy. Uncrusify doesn't need 'python version'